includes/class-wc-payment-information.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Class WC_Payment_Information
+ *
+ * @package WooCommerce\Payments
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Mostly a wrapper containing information on a single payment.
+ */
+class WC_Payment_Information {
+	/**
+	 * The ID of the payment method used for this payment.
+	 *
+	 * @var string
+	 */
+	private $payment_method;
+
+	/**
+	 * The payment token used for this payment.
+	 *
+	 * @var WC_Payment_Token/NULL
+	 */
+	private $token;
+
+	/**
+	 * Indicates whether this payment was made using a saved card.
+	 *
+	 * @var bool
+	 */
+	private $is_saved_method;
+
+	/**
+	 * Indicates whether the payment is merchant-initiated (true) or customer-initiated (false).
+	 *
+	 * @var bool
+	 */
+	private $off_session;
+
+	/**
+	 * Indicates whether this is a one-off payment (false) or the first installment of a recurring payment (true).
+	 *
+	 * @var bool
+	 */
+	private $is_recurring;
```

Comment:
@DanReyLop this seems to have been documented incorrectly in the initial push? The doc comment indicated that this parameter would only be true when we're processing the **first instalment** of a recurring payment, but this parameter is also true for renewal payments.

I'm assuming the doc comment should be

>Indicates whether this is a one-off payment (false) or part of a set of recurring payments [subscriptions] (true).

Reply:
This is outdated since we've phased this out of the `Payment_Information` class.